### PR TITLE
feature/9816-EvenMoreIconMigrations

### PR DESCRIPTION
**Pull request by Sparowhawk**
## Description of Change
Photo add(upload photos screen)
Photo Upload(veteran status screen, not used)
Left Chevron (pagination, descriptive back button, back button usage wasn't used so removed Icon all together from that component)
Right Chevron(Lists, nametag, link row, pagination, inline text)

## Screenshots/Video
Screenshot 2024-10-30 at 4 07 35 PM
Screenshot 2024-10-30 at 4 07 40 PM
Screenshot 2024-10-30 at 4 08 21 PM
Screenshot 2024-10-30 at 4 11 23 PM
Screenshot 2024-10-30 at 4 16 01 PM
Screenshot 2024-10-31 at 9 27 36 AM
Screenshot 2024-10-31 at 9 45 29 AM

## Testing
yarn test, VQA in slack with Lauren

- [x] Tested on iOS 
- [x] Tested on Android 

## Reviewer Validations
Migrate icons over and VQA with Lauren to make any adjustments

## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [x] PR is connected to issue(s)
- [x] Tests are included to cover this change (when possible)
- [x] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [x] No secrets or API keys are checked in
- [x] All imports are absolute (no relative imports)
- [x] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

File: VAMobile/src/components/BackButton.tsx [hunk 1]
```diff
@@ -59,8 +55,6 @@ export const BackButton: FC<BackButtonProps> = ({
     return null
   }
 
-  const chevron = showCarat ? <VAIcon mt={1} name={'ChevronLeft'} fill="backButton" testID="BackButtonCarat" /> : <></>
```
Comment (Sparowhawk) [inline VAMobile/src/components/BackButton.tsx:62]: with us using our own screen templates, the back button in headerStyles is technically not used, but in the off chance we do forget to have a template applied to a screen I opted to keep the component and just remove the icon.

Comment (Sparowhawk) [reply VAMobile/src/components/BackButton.tsx:62]: see prior comment in back button

File: VAMobile/src/components/InlineTextWithIcons.test.tsx [hunk 2]
```diff
@@ -61,6 +61,6 @@ context('InlineTextWithIcons', () => {
   it('renders icons', () => {
     expect(screen.getByTestId('Unread')).toBeTruthy()
     expect(screen.getByTestId('PaperClip')).toBeTruthy()
-    expect(screen.getByTestId('ChevronRight')).toBeTruthy()
```
Comment (Sparowhawk) [inline VAMobile/src/components/InlineTextWithIcons.test.tsx:64]: updated test here so I could remove the Chevron SVG

**Review by alexandec — DISMISSED** [review]

**Review by alexandec — DISMISSED** [review]

**Review by theodur — COMMENTED** [review]
Noticed some of the unit tests are failing

**Review by dumathane — APPROVED** [review]

**Review by theodur — COMMENTED** [review]
Linting and unit tests are still failing

**Review by dumathane — APPROVED** [review]

**Review by rbontrager — APPROVED** [review]

**Pull request merged**